spec/features/user_visits_book_show_spec.rb

```diff
@@ -0,0 +1,67 @@
+describe "User visits book show page" do
+  context "sees book title and reviews" do
+    before(:all) do
+      @book = Book.create(title: "Bible")
+      @user_1 = User.create(name: "KenM")
+      @user_2 = User.create(name: "Max")
+
+      @review_1 = @book.reviews.create(body: "Best story book ever",
+                                   user_id: @user_1.id,
```

Comment:
You can also pass in arguments like this `user: @user` instead of  `user_id: @user.id`.  Both are completely acceptable the second is just less to type and maybe slightly easier to read.